packages/components/src/templates/next/layouts/Homepage/Homepage.stories.tsx

```diff
@@ -297,6 +297,41 @@ const generateArgs = ({
         buttonLabel: "Primary CTA",
         buttonUrl: "/",
       },
+      {
+        type: "infopic",
+        title:
+          "Explore your great neighbourhood with us can’t stretch all the way so this needs a max width",
+        description:
+          "They will try to close the door on you, just open it. Lion! The other day the grass was brown, now it’s green because I ain’t give up. Never surrender.",
+        imageAlt: "alt",
+        imageSrc: "https://placehold.co/200x200",
+        buttonLabel: "Primary CTA",
+        buttonUrl: "[resource:1:1]",
+      },
+      {
+        variant: "full",
+        type: "infopic",
+        title:
+          "Explore your great neighbourhood with us can’t stretch all the way so this needs a max width",
+        description:
+          "They will try to close the door on you, just open it. Lion! The other day the grass was brown, now it’s green because I ain’t give up. Never surrender.",
+        imageAlt: "alt",
+        imageSrc: "https://placehold.co/200x200",
+        buttonLabel: "Primary CTA",
+        buttonUrl: "/",
+      },
+      {
+        variant: "full",
+        type: "infopic",
+        title:
+          "Explore your great neighbourhood with us can’t stretch all the way so this needs a max width",
+        description:
+          "They will try to close the door on you, just open it. Lion! The other day the grass was brown, now it’s green because I ain’t give up. Never surrender.",
+        imageAlt: "alt",
+        imageSrc: "https://placehold.co/200x200",
+        buttonLabel: "Primary CTA",
+        buttonUrl: "/",
+      },
```

Comment:
curious - why need both ah?

Reply:
we want to check that: 
1. full/block will still have the left/right stacking behaviour
2. we want to check that full/full wil showt he stacking correctly also